server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/RemoveRegexMappingRoutes.java

```diff
@@ -0,0 +1,82 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.webadmin.routes;
+
+import static spark.Spark.halt;
+
+import javax.inject.Inject;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+
+import org.apache.james.rrt.api.RecipientRewriteTable;
+import org.apache.james.rrt.api.RecipientRewriteTableException;
+import org.apache.james.rrt.lib.MappingSource;
+import org.apache.james.webadmin.Constants;
+import org.apache.james.webadmin.Routes;
+import org.apache.james.webadmin.utils.ErrorResponder;
+import org.eclipse.jetty.http.HttpStatus;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import io.swagger.annotations.Api;
+import spark.HaltException;
+import spark.Request;
+import spark.Response;
+import spark.Service;
+
+@Api(tags = "Remove Regex")
+@Path(RemoveRegexMappingRoutes.BASE_PATH)
+@Produces(Constants.JSON_CONTENT_TYPE)
+public class RemoveRegexMappingRoutes implements Routes {
```

Comment:
you should write your route in the `RegexMappingRoutes` class and avoid to create new route class if possible

Reply:
Because the PR #2526 still receives comments, I will wait until #2526 is fully accepted, then,
maybe in this pull request, I will add `RegexMappingRoutes` implementation, but with `RemoveRegexMappingRoutes` implementation integrated ? 
Also, is it OK if `class` name would be changed from `RemoveRegexMappingRoutes` => `RegexMappingRoutes` (possibly confuse reviewers, they already review `RegexMappingRoutes` in #2526 ) ?